substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1264,6 +1264,16 @@ mod tests {
 mod remote_tests {
 	use super::test_prelude::*;
 	use std::os::unix::fs::MetadataExt;
+	use std::env;
+
+	static TRANSPORT_URL: once_cell::sync::Lazy<String> = once_cell::sync::Lazy::new(|| {
+        env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string())
```

Comment:
You can just use a function:  
```rust
fn endpoint() -> String {
    env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string())
}
```
the computation is not complicated so no need to make store it.

Reply:
Done. Refactored to use a function instead.